examples/common/builders.py

```diff
@@ -67,9 +71,9 @@ def build_optimizer(cfg, model):
                               weight_decay=cfg.weight_decay)
     elif cfg.name == 'decoupled_lionw':
         return DecoupledLionW(model.parameters(),
-                    lr=cfg.lr,
-                    betas=cfg.betas,
-                    weight_decay=cfg.weight_decay)
+                              lr=cfg.lr,
+                              betas=cfg.betas,
+                              weight_decay=cfg.weight_decay)
```

Comment:
sneaking in a little extra linting I see 😄

Reply:
i just do what the linter says :D